draft-ietf-jsonpath-base.md

```diff
@@ -421,6 +421,10 @@ type errors when pondering well-formedness and validity, while
 resource depletion and related errors are comparable to 500 type
 errors.)
 
+The JSON value the JSONPath query is applied to is, by definition, valid JSON.
```

Comment:
I'm not sure "valid JSON" is defined.  "a valid JSON value" clarifies this and also exposes the redundancy of this statement.

```suggestion
The JSON value the JSONPath query is applied to is, by definition, a valid JSON value.
```